src/doc_tree/mod.rs

```diff
@@ -9,5 +9,10 @@
 ///
 /// The DocTree can be consumed by a `DocGenerator` to produce
 /// documentation comments.
-#[allow(unused)]
 pub struct DocTree;
+
+impl From<syn::File> for DocTree {
+    fn from(_file: syn::File) -> Self {
+        todo!()
+    }
+}
```

Comment:
Looks good to me!

Though I do like to have an issue associated with all todo!() items.

Do you think you could link (and create if needed) issues for these todos?